code/modules/modular_computers/file_system/programs/system/file_browser.dm

```diff
@@ -60,21 +92,40 @@
 		if(!HDD)
 			return 1
 		var/datum/computer_file/F = HDD.find_file_by_name(href_list["PRG_clone"])
+		var/checkpass = ""
+		var/currentpass = F.password
```

Comment:
You're accessing `F.password` without checking if F actually exists. Line 97 checks if F exists and exits if it doesn't. So this has to be below it.